src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      const propertiesKeys = Object.keys(transforms[i].properties);
+
+      for (let j = 0; j < propertiesKeys.length; j++) {
+        state[propertiesKeys[j]] = transforms[i].properties[propertiesKeys[j]];
+      }
+    } else if (transforms[i].operation === 'clear') {
+      const stateKeys = Object.keys(state);
+
+      for (let k = 0; k < stateKeys.length; k++) {
+        delete state[stateKeys[k]];
+      }
```

Comment:
```suggestion
      for (const key in state) {
        delete state[key];
      }
```
Why not use 'for...in' loop here and just iterate through the keys of state?
In such case you will not need to create `const stateKeys = Object.keys(state);` and it will make code clearer